libs/store/src/lib/infrastructure/persistence/domain/state-handler.spec.ts

```diff
@@ -25,15 +21,18 @@ interface ReqDTO {
     lineItems: Array<LineItem>;
 }
 
-class Money extends BaseValue<MoneyDTO> {
-    constructor(value?: MoneyDTO) {
-        super(value);
+class Money extends BaseValue<MoneyDTO> implements MoneyDTO {
+    amount: number;
+    currency: string;
+
+    constructor(dto: MoneyDTO) {
```

Comment:
how are. you using DTO here? I dont think its used at all right ?

Reply:
I removed this constructor here so it's not required

Reply:
Parent class constructor is used by default